dev/create-release/release-build.sh

```diff
@@ -84,7 +84,7 @@ GIT_REF=${GIT_REF:-master}
 
 RELEASE_STAGING_LOCATION="https://dist.apache.org/repos/dist/dev/spark"
 
-GPG="gpg -u $GPG_KEY --no-tty --batch"
+GPG="gpg -u $GPG_KEY --no-tty --batch --pinentry-mode loopback"
```

Comment:
I noticed the failures on Jenkins job `spark-master-package`: https://amplab.cs.berkeley.edu/jenkins/view/Spark%20Packaging/job/spark-master-package/2275/console

It seems to be related to this change.

```
gpg: invalid option "--pinentry-mode"
Deleting credential directory /home/jenkins/workspace/spark-master-package/spark-utils/new-release-scripts/jenkins/jenkins-credentials-tVKlQBOF
Build step 'Execute shell' marked build as failure
[WS-CLEANUP] Deleting project workspace...[WS-CLEANUP] done
Finished: FAILURE
```

cc @dongjoon-hyun @dbtsai

Reply:
Opened https://issues.apache.org/jira/browse/SPARK-27984 to track it.